src/components/loadable.cjsx

```diff
@@ -21,14 +23,17 @@ module.exports = React.createClass
     renderError: React.PropTypes.func.isRequired
     renderBug: React.PropTypes.func.isRequired
 
+  getInitialState: ->
+    refreshButton: false
+
   getDefaultProps: ->
 
     # Enables a renderStatus prop function with a component other than a div
-    renderLoading: ->
-      <div className='-loading'>Loading...</div>
+    renderLoading: (refreshButton) ->
```

Comment:
I found using _.delay to not work consistently within the render statuses, and using the css delay worked smoothly.  That way there is no timeout that could break render, it just shows a button.